docs/reference/indices/put-mapping.asciidoc

```diff
@@ -75,11 +62,112 @@ you only want to rename a field, consider adding an <<alias, `alias`>> field.
 
 // end::put-field-mapping-exceptions[]
 
+
+[[put-mapping-api-path-params]]
+==== {api-path-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=index]
++
+To update the mapping of all indices, omit this parameter or use a value of
+`_all`.
+
+
+[[put-mapping-api-query-params]]
+==== {api-query-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=allow-no-indices]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=expand-wildcards]
++
+Defaults to `open`.
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=include-type-name]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=index-ignore-unavailable]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=timeoutparms]
+
+
+[[put-mapping-api-request-body]]
+==== {api-request-body-title}
+
+`properties`::
++
+--
+(Required, <<mapping,mapping object>>) Mapping for a field. For new
+fields, this mapping can include:
+
+* Field name
+* <<field-datatypes,Field datatype>>
+* <<mapping-params,Mapping parameters>>
+
+For existing fields, see <<updating-field-mappings>>.
+--
+
+
+[[put-mapping-api-example]]
+==== {api-examples-title}
+
+[[put-field-mapping-api-basic-ex]]
+===== Example with index setup
+
+The put mapping API requires an existing index. The following
+<<indices-create-index, create index>> API request creates the `publications`
+index with no mapping.
+
+[source,js]
+----
+PUT /publications
+----
+// CONSOLE
+
+The following put mapping API request adds `title`, a new <<text,`text`>> field,
+to the `publications` index.
+
+[source,js]
+----
+PUT /publications/_mapping
+{
+  "properties": {
+    "title":  { "type": "text"}
+  }
+}
+----
+// CONSOLE
+// TEST[continued]
+
+[[put-mapping-api-multi-ex]]
+===== Multiple indices
+
+The PUT mapping API can be applied to multiple indices with a single request.
+For example, we can update the `twitter-1` and `twitter-2` mappings at the same time:
+
+[source,js]
+--------------------------------------------------
+# Create the two indices
+PUT /twitter-1
+PUT /twitter-2
+
+# Update both mappings
+PUT /twitter-1,twitter-2/_mapping <1>
+{
+  "properties": {
+    "user_name": {
+      "type": "text"
+    }
+  }
+}
+--------------------------------------------------
+// CONSOLE
+// TEST[setup:twitter]
+
+<1> Note that the indices specified (`twitter-1,twitter-2`) follows <<multi-index,multiple index names>> and wildcard format.
+
 For example:
 
 [source,js]
 -----------------------------------
-PUT my_index <1>
+PUT /my_index <1>
```

Comment:
It is strange to have "For example:" sentence on line 166 and an example below in this section of the document. Before it was showing an example for updating mapping of the existing documents. Now it ended up in the section "Multiple indices".  I think we need to move these examples under the section "Restrictions for existing fields"

Reply:
Thanks for catching this @mayya-sharipova. It looks like I mangled the attempt to move this under the description section.

With35e9d82, I moved info about updating existing mappings back to the examples section.